providers/google-chat/README.md

```diff
@@ -0,0 +1,11 @@
+# Novu GoogleChat Provider
+
+A GoogleChat chat provider library for [@novu/node](https://github.com/novuhq/novu)
+
+## Usage
+
+```javascript
+import { GoogleChatProvider } from '@novu/google-chat';
+
+const provider = new GoogleChatProvider({});
```

Comment:
The angle brackets should not be here, should just be ().

Reply:
Thanks @Cliftonz, I updated it.